Update egress gateway docs (#1067)
tigera/docs: calico-cloud/networking/egress/egress-gateway-azure.mdx

### Create Azure Route Server

> Deploy Azure [Route Server](https://learn.microsoft.com/en-us/azure/route-server/) in the VNet that routes about

Reviewer: ```suggestion
Deploy Azure [Route Server](https://learn.microsoft.com/en-us/azure/route-server/) in the VNet (hub or spoke VNet) that routes
```

> egress addresses need to be advertised to. This can be either a hub or spoke VNet. Then add the chosen route reflectors

Reviewer: ```suggestion
egress addresses for advertisement. Then, add the applicable route reflectors as peers to the server. 
```

EOF
```

> Even with BGP filter, {{prodname}} still advertise pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set

Reviewer: ```suggestion
Even with BGP filter, {{prodname}} still advertises pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set
```

> Even with BGP filter, {{prodname}} still advertise pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set

Reviewer: ```suggestion
Even with the BGP filter, {{prodname}} still advertises pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set
```
